docs/user-interface/images/splashscreens.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Splash screens"
+description: ".NET Multi-platform App UI (.NET MAUI) splash screens can be displayed on Android and iOS when an app is launched, while the app's initialization process completes."
+ms.date: 08/19/2021
+---
+
+# Splash screens
+
+<!-- Todo: Move this content into the images doc, once it migrates over? -->
+
+On Android and iOS, .NET Multi-platform App UI (.NET MAUI) apps can display a splash screen while their initialization process completes. The splash screen is displayed immediately when an app is launched, providing immediate feedback to users while app resources are initialized. Once the app is ready for interaction, the splash screen is dismissed.
+
+Splash screens are a composite of an image and a background color. The standard platform image formats are supported, including Scalable Vector Graphics (SVG) files.
+
+> [!TIP]
+> The SVG format is the recommended image format for .NET MAUI splash screens.
+
+.NET MAUI splash screens can be added to your app project by dragging an image into the _Resources\Images_ folder of the project, and setting the build action of the image to **MauiSplashScreen** in the **Properties** window. This creates a corresponding entry in your project file:
+
+<!-- Todo: The template currently puts the splash screen in Resources, not Resources\Images -->
+
+```xml
+<MauiSplashScreen Include="Resources\Images\splashscreen.svg" />
+```
+
+Splash screen files names must be lowercase, start and end with a letter character, and contain only alphanumeric characters or underscores.
```

Comment:
You might say "to comply with Android resource naming rules" and then maybe we have a link to an existing doc?

If not here is a Google one: https://developer.android.com/guide/topics/resources/providing-resources